tpchgen-cli/src/plan.rs

```diff
@@ -231,21 +249,20 @@ impl OutputSize {
                 Table::Region => 151,
                 Table::Part => 70,
                 Table::Supplier => 164,
-                Table::Partsupp => 140,
+                Table::Partsupp => 141 * 4, // needed to match observed size
```

Comment:
I changed this due when reviewing the output of  the row group size tests and saw the row groups for partsupp were like 4x larger than the target for some reason